sickbeard/providers/torrentz.py

```diff
@@ -90,7 +90,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                             if item.category and 'tv' not in item.category.get_text(strip=True):
                                 continue
 
-                            title = item.title.text.rsplit(' ', 1)[0].replace(' ', '.')
+                            title = item.title.text.rsplit(' ', 1)[0].replace('x264 ', 'x264-').replace('h264 ', 'h264-').replace('h 264 ', 'h264-').replace('xvid ', 'xvid-').replace(' ', '.')
```

Comment:
Yes because we only can be sure of scene releases to replace, currently  they only release  in x264, h264 and some xvid, but not for the tv-264 category  ofc. Everything  else if exists it's an internal and aren't nuked/propered
Plus what I said before